CMakeLists.txt

```diff
@@ -353,7 +353,7 @@ if(NRN_ENABLE_INTERVIEWS)
     get_target_property(IV_INCLUDE_DIR interviews INTERFACE_INCLUDE_DIRECTORIES)
   else()
     nrn_add_external_project(iv)
-    include_directories(external/iv/src/include)
+    include_directories(SYSTEM external/iv/src/include)
```

Comment:
Just curious. What is the rationale. I see from cmake doc that
> If the SYSTEM option is given, the compiler will be told the directories are meant as system include directories on some platforms. Signalling this setting might achieve effects such as the compiler skipping warnings, or these fixed-install system files not being considered in dependency calculations - see compiler docs.

Is this mostly for skipping warnings? Co-development  of IV and nrn is rare, so not being considered in dependency calculations isn't a strong negative.  The "some platforms" caveat makes it all foggy to me.

Reply:
I got a lot of warning from IV but I think we don't want to fix them ,right? as it is an external dependency, so we don't see the **internal** warning of IV. If you misuse the public API of IV inside nrn it will produce warning.

Reply:
Ok. That sounds right. I might someday work on that to make it -Wall clean, but it's low priority.